<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MYWindow</class>
 <widget class="QMainWindow" name="MYWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>941</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MYWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QwtPlot" name="qwtPlot">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>169</y>
      <width>1011</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1011</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_connect">
       <property name="text">
        <string>CONNECT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_config">
       <property name="text">
        <string>CONFIG</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_start">
       <property name="text">
        <string>START</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_stop">
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_disconnect">
       <property name="text">
        <string>DISCONNECT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>1011</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radio_save">
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_display">
       <property name="text">
        <string>display</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_startdisplay">
       <property name="text">
        <string>START_DISPLAY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_stopdisplay">
       <property name="text">
        <string>STOP_DISPLAY</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>1011</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>level</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>info</string>
     </property>
    </column>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>689</y>
      <width>20</width>
      <height>171</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
